ui/raidboss/data/07-dt/raid/r4n.ts

```diff
@@ -1,14 +1,274 @@
+      id: 'R4N Headmarker Wicked Jolt Tankbuster',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.tankBusterLine, capture: true },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'R4N Wrath of Zeus',
+      type: 'StartsUsing',
+      netRegex: { id: '92C7', source: 'Wicked Thunder', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'R4N Sidewise Spark Go Left',
+      type: 'StartsUsing',
+      netRegex: { id: ['92BC', '92BE'], source: 'Wicked Thunder', capture: false },
+      response: Responses.goLeft(),
+    },
+    {
+      id: 'R4N Sidewise Spark Go Right',
+      type: 'StartsUsing',
+      netRegex: { id: ['92BD', '92BF'], source: 'Wicked Thunder', capture: false },
+      response: Responses.goRight(),
+    },
+    {
+      id: 'R4N Left Roll',
+      type: 'Ability',
+      netRegex: { id: '92AC', source: 'Wicked Thunder', capture: false },
+      response: Responses.goWest(),
+    },
+    {
+      id: 'R4N Right Roll',
+      type: 'Ability',
+      netRegex: { id: '92AB', source: 'Wicked Thunder', capture: false },
+      response: Responses.goEast(),
+    },
+    {
+      id: 'R4N Threefold Blast Initializer',
+      type: 'StartsUsing',
+      netRegex: { id: ['92AD', '92B0'], source: 'Wicked Thunder', capture: false },
+      run: (data) => data.expectedBlasts = 3,
+    },
+    {
+      id: 'R4N Fourfold Blast Initializer',
+      type: 'StartsUsing',
+      netRegex: { id: ['9B4F', '9B55'], source: 'Wicked Thunder', capture: false },
+      run: (data) => data.expectedBlasts = 4,
+    },
+    {
+      id: 'R4N Fivefold Blast Initializer',
+      type: 'StartsUsing',
+      netRegex: { id: ['9B56', '9B57'], source: 'Wicked Thunder', capture: false },
+      run: (data) => data.expectedBlasts = 5,
+    },
+    {
+      id: 'R4N XFold Blast Collector',
+      type: 'GainsEffect',
+      netRegex: { effectId: 'B9A', count: Object.values(effectB9AMap), capture: true },
+      condition: (data, matches) => {
+        const count = matches.count;
+
+        if (!isEffectB9AValue(count))
+          return false;
+        data.storedBlasts.push(count);
+
+        return data.storedBlasts.length >= data.expectedBlasts;
+      },
+      durationSeconds: (data) => {
+        if (data.expectedBlasts === 3)
+          return 14.4;
+        if (data.expectedBlasts === 4)
+          return 18.9;
+        return 23.2;
+      },
+      infoText: (data, _matches, output) => {
+        const expectedBlasts = data.expectedBlasts;
+        data.expectedBlasts = 3;
+        const storedBlasts = data.storedBlasts;
+        data.storedBlasts = [];
+        switch (expectedBlasts) {
+          case 3:
+            return output.three!({
+              first: b9aValueToNorthSouth(storedBlasts[0]),
+              second: b9aValueToNorthSouth(storedBlasts[1]),
+              third: b9aValueToNorthSouth(storedBlasts[2]),
+            });
+          case 4:
+            return output.four!({
```

Comment:
I think these should be `output[b9aValueToNorthSouth(storedBlasts[0])!()` if you want to use the actual output strings and not just 'north' and 'south' literals.

But I think you could probably simplify this a bit and get rid of the entire switch by just doing:
```
        const comboOutput = storedBlasts.map((v) => output[b9aValueToNorthSouth(v)]!()).join(' => ');
        return output.combo!({ dirs: comboOutput });
```
And then in `outputStrings`, you can remove `three`, `four`, and `five`, and just do:

```
      outputStrings: {
        south: Outputs.south,
        north: Outputs.north,
        unknown: Outputs.unknown,
        combo: {
          en: '${dirs}',
        },
      },
```

Reply:
I did not want to hardcode the arrows and not allow other languages to change the separator.

Reply:
Fair.  In that case, I think you could do the separator as its own prop on `outputStrings` and use it in the `join()`.  But this was really just a stylistic comment, so feel free to ignore (although we still should use `output[]` on each of the props).

Reply:
Yeah, if we do simplify it down to a `join`, we really want the separator defined per-language. It might be worth adding to `outputs.ts`?

Reply:
I think that the changes I've made make sense and line up with what you were proposing?